src/data/aktivitetskrav/useCreateAktivitetskrav.ts

```diff
@@ -0,0 +1,21 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { ISAKTIVITETSKRAV_ROOT } from "@/apiConstants";
+import { post } from "@/api/axios";
+import { aktivitetskravQueryKeys } from "@/data/aktivitetskrav/aktivitetskravQueryHooks";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+
+export const useCreateAktivitetskrav = () => {
+  const personident = useValgtPersonident();
+  const queryClient = useQueryClient();
+  const path = `${ISAKTIVITETSKRAV_ROOT}/aktivitetskrav`;
+  const postAktivitetskrav = () => post(path, {}, personident);
+
+  return useMutation({
+    mutationFn: postAktivitetskrav,
+    onSuccess: () => {
+      return queryClient.invalidateQueries(
+        aktivitetskravQueryKeys.aktivitetskrav(personident)
+      );
+    },
+  });
```

Comment:
Hva med å bare bruke responsen fra post kallet? Er det en grunn til å ta en ekstra roundtrip? 😊

Reply:
Ja, det hadde nok godt an å gjort setQueryData her :thinking: Det enkleste da hadde kanskje vært om POST-kallet hadde returnert en respons på samme format som GET-kallet så vi slipper å ha en egen type for POST-responsen i frontend som må passe inn i resultate fra GET-kallet. Vi kan jo se på det i en ny PR kanskje.